### PR TITLE
give create ai powered insight

### PR DESCRIPTION
**Pull request by nikhiljha**
evaluate new technologies at 10x speed

### CODE REVIEW

File: ircbot/plugin/insight [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""Give automated insight on new technology"""
+
+
+def register(bot):
+    bot.listen(r'^!insight (.+)', insight)
+
+
+def show_comic(_, msg):
```
Comment (nattofriends) [inline ircbot/plugin/insight:8]: Comic?

**Review by nattofriends — COMMENTED** [review]
Might want a file extension

File: ircbot/plugin/insight (hunk 1 above)
Comment (nikhiljha) [reply ircbot/plugin/insight]: oops

**Review by nattofriends — APPROVED** [review]

**Review by jvperrin — COMMENTED** [review]
Maybe before we rush to adopt AI powered insight we should stop to consider the consequences of blithely giving this technology such a central position in our lives.

File: ircbot/plugin/insight.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"""Give automated insight on new technology"""
+
+
+def register(bot):
+    bot.listen(r'^!insight (.+)', insight)
+
+
+def insight(_, msg):
+    """Modern AI based technology insight."""
+    s = "Maybe before we rush to adopt {0} we should stop to consider the consequences of blithely giving this technology such a central position in our lives."
```
Comment (jvperrin) [inline ircbot/plugin/insight.py:10]: This line is too long and is making the pre-commit checks fail, maybe split it into multiple lines?

Comment (nikhiljha) [reply ircbot/plugin/insight.py:10]: splitting it onto multiple lines makes it look worse :(

something something society has progressed beyond the need for line limits

Comment (nattofriends) [reply ircbot/plugin/insight.py]: This may afford you no solace but you can use adjacent string literals to carry out this task.

**Pull request merged**